src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable react/jsx-one-expression-per-line */
+export function Person({ person }) {
+  let marriage = 'I am not married';
+
+  if (person.isMarried && person.sex === 'm') {
+    marriage = `${person.partnerName} is my wife`;
+  }
+
+  if (person.isMarried && person.sex === 'f') {
+    marriage = `${person.partnerName} is my husband`;
+  }
+
```

Comment:
Simplify as you did before
you can write it with one `if` statement and `ternary` operator.